packages/ui/src/components/editor/input/Vector3/index.tsx

```diff
@@ -124,13 +124,13 @@ export const Vector3Input = ({
   const vz = value.z
 
   return (
-    <div className="flex flex-wrap justify-start gap-1.5">
+    <div className="flex flex-row flex-nowrap justify-start gap-1.5">
```

Comment:
why was `flex-nowrap` added?
on smaller screens the vector input is overflowing
![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/af4e5198-3e9d-44df-a522-14efe6e3fca8)

Reply:
resolved